int32 resultsCount
ArtResult[] results